sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -705,3 +705,13 @@
 .formio-component-file ul li.list-group-item {
   border: 1px solid rgba(0, 0, 0, 0.125) !important;
 }
+
+// Removes the required sign from the input fields with .no-required-field-sign.
+.no-required-field-sign label.field-required:after {
+  content: none;
```

Comment:
Should we force here with `!important` as well?

Reply:
No need for that one as there wasn't a class preceding and overwriting it.